File: Statistics/lesson1/sources/Lesson1VBNET/Ball.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace Lesson1CS
    Class Ball
        Private width As Integer
        Private height As Integer
        Private x As Integer
        Private y As Integer
        Private stepx As Integer
        Private stepy As Integer

        Public Sub New(ByVal w As Integer, ByVal h As Integer, ByVal xVar As Integer, ByVal yVar As Integer, ByVal stepxVar As Integer, ByVal stepyVar As Integer)
            width = w
            height = h
            x = xVar
            y = yVar
            stepx = stepxVar
            stepy = stepyVar
        End Sub

        Public Function GetWidth() As Integer
            Return width
        End Function

        Public Function GetHeight() As Integer
            Return height
        End Function

        Public Function GetX() As Integer
            Return x
        End Function

        Public Function GetY() As Integer
            Return y
        End Function

        Public Function GetStepX() As Integer
            Return stepx
        End Function

        Public Function GetStepY() As Integer
            Return stepy
        End Function

        Public Sub SetWidth(ByVal w As Integer)
            width = w
        End Sub

        Public Sub SetHeight(ByVal h As Integer)
            height = h
        End Sub

        Public Sub SetX(ByVal xVar As Integer)
            x = xVar
        End Sub

        Public Sub SetY(ByVal yVar As Integer)
            y = yVar
        End Sub

        Public Sub SetStepX(ByVal stepxVar As Integer)
            stepx = stepxVar
        End Sub

        Public Sub SetStepY(ByVal stepyVar As Integer)
            stepy = stepyVar
        End Sub
    End Class
End Namespace

File: Statistics/lesson1/sources/Lesson1CS/Lesson1CS/Ball.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lesson1CS
{
    class Ball
    {
        private int width;
        private int height;
        private int x;
        private int y;
        private int stepx;
        private int stepy;
        

        public Ball(int w, int h, int xVar, int yVar, int stepxVar, int stepyVar)
        {
            width = w;
            height = h;
            x = xVar;
            y = yVar;
            stepx = stepxVar;
            stepy = stepyVar;
        }

        public int GetWidth()
        {
            return width;
        }

        public int GetHeight()
        {
            return height;
        }

        public int GetX()
        {
            return x;
        }

        public int GetY()
        {
            return y;
        }

        public int GetStepX()
        {
            return stepx;
        }

        public int GetStepY()
        {
            return stepy;
        }

        public void SetWidth(int w)
        {
            width = w;
        }

        public void SetHeight(int h)
        {
            height = h;
        }

        public void SetX(int xVar)
        {
            x= xVar;
        }

        public void SetY(int yVar)
        {
            y = yVar;
        }

        public void SetStepX(int stepxVar)
        {
            stepx = stepxVar;
        }

        public void SetStepY(int stepyVar)
        {
            stepy = stepyVar;
        }
    }
}
